includes/Checker/Check_Categories.php

```diff
@@ -46,13 +40,35 @@ public static function get_categories() {
 	 * @return array An array of category labels.
 	 */
 	public static function get_category_labels() {
-		return array(
+		return self::get_check_categories();
+	}
+
+	/**
+	 * Returns an array of check categories.
+	 *
+	 * @since 1.0.2
+	 *
+	 * @return array An array of check categories.
+	 */
+	public static function get_check_categories() {
+		$default_categories = array(
 			self::CATEGORY_GENERAL       => __( 'General', 'plugin-check' ),
 			self::CATEGORY_PLUGIN_REPO   => __( 'Plugin Repo', 'plugin-check' ),
 			self::CATEGORY_SECURITY      => __( 'Security', 'plugin-check' ),
 			self::CATEGORY_PERFORMANCE   => __( 'Performance', 'plugin-check' ),
 			self::CATEGORY_ACCESSIBILITY => __( 'Accessibility', 'plugin-check' ),
 		);
+
+		/**
+		 * Filters the check categories.
+		 *
+		 * @since 1.0.2
+		 *
+		 * @param array $default_categories An array of check categories.
```

Comment:
Nit: let’s explain that it‘s an associative array

Reply:
Sure. Would it make more sense following array structure like `block_categories_all`? It would be more future proof and we can easily add attributes for each category in the future if needed.

Reply:
I don't think we need that. If we ever need more complex data structure we can still add it later.

What I mean here is something like 
```suggestion
		 * @param array<string, string> $default_categories Associative array of category slugs to labels.
```